### PR TITLE
Bugfix/live 8799 fix storage available

### PR DESCRIPTION
**Pull request by aussedatlo**
### 📝 Description

On LLD with Stax only, the size of free memory was incorrect, since the reserved space for stax image was not accounted for.
Now, the application show the correct amount of free storage.

### ❓ Context

- **Impacted projects**: `LLD` `common`
- **Linked resource(s)**: [LIVE-8799]

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-8799]: https://ledgerhq.atlassian.net/browse/LIVE-8799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ledger-live-common/src/apps/logic.ts [hunk 1]
```diff
@@ -437,12 +437,13 @@ export function getBlockSize(state: State): number {
...
 // calculate if a given state (typically a predicted one) is out of memory (meaning impossible to reach with a device)
 export const isOutOfMemoryState = (state: State): boolean => {
+  const { customImageBlocks } = state;
   const blockSize = getBlockSize(state);
   const totalBytes = state.deviceModel.memorySize;
   const totalBlocks = Math.floor(totalBytes / blockSize);
   const osBytes = (state.firmware && state.firmware.bytes) || 0;
   const osBlocks = Math.ceil(osBytes / blockSize);
-  const appsSpaceBlocks = totalBlocks - osBlocks;
+  const appsSpaceBlocks = totalBlocks - osBlocks - customImageBlocks;
```
Comment (ofreyssinet-ledger) [inline libs/ledger-live-common/src/apps/logic.ts:446]: shouldn't this `isOutOfMemoryState` function rely on `distribute` to ensure that all these calculations are only implement in one place and avoid the apparition of such discrepancies 
(in this case the computation of `appsSpaceBlocks` was implemented in two places and at some point there was a divergence) ?

Comment (aussedatlo) [reply libs/ledger-live-common/src/apps/logic.ts:446]: Good point here, I'll do it 👍

**Review by ofreyssinet-ledger — APPROVED** [review]
nice fix, lgtm 👍

**Review by smartine-ledger — APPROVED** [review]
seems ok 👍

**Pull request closed without merging**